written by Mike Frysinger

https://sourceforge.net/tracker/?func=detail&aid=3132051&group_id=5616&atid=305616

The top level makefile ignores build/install errors in subdirs which makes
packaging a pain to verify.

--- a/Makefile
+++ b/Makefile
@@ -31,11 +31,11 @@
 CFLAGS= -O2 -W -Wall
 
 all:
-	@-(set -e ; for d in $(DIRS) ; do $(MAKE) $(MAKE_OUTPUT) -C $$d $@ ; done)
+	@(set -e ; for d in $(DIRS) ; do $(MAKE) $(MAKE_OUTPUT) -C $$d $@ ; done)
 
 
 install:
-	@-(set -e ; for d in $(DIRS) ; do $(MAKE) $(MAKE_OUTPUT) -C $$d $@ ; done)
+	@(set -e ; for d in $(DIRS) ; do $(MAKE) $(MAKE_OUTPUT) -C $$d $@ ; done)
 
 
 clean:
